iam-bootstrap/bootstrap-1.json

```diff
@@ -91,6 +91,19 @@
                 "*"
             ]
         },
+        {
+            "Sid": "ACMUngated",
+            "Effect": "Allow",
+            "Action": [
+                "acm:RequestCertificate",
+                "acm:DescribeCertificate",
+                "acm:GetCertificate",
+                "acm:ListTagsForCertificate"
```

Comment:
Dont we need [DeleteCertificate](https://docs.aws.amazon.com/acm/latest/APIReference/API_DeleteCertificate.html)